packages/gatsby-transformer-webapi/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@commercetools-docs/gatsby-transformer-webapi",
+  "private": true,
+  "license": "MIT",
+  "version": "1.0.0-alpha.0",
+  "description": "Exposes fully resolved and validated RAML API Specifications on the gatsbyJS GraphQL",
+  "author": "commercetools GmbH <support@commercetools.com>",
+  "homepage": "https://github.com/commercetools/commercetools-docs-kit/tree/master/packages/gatsby-transformer-webapi#readme",
+  "keywords": [
+    "gatsby",
+    "gatsby-plugin",
+    "raml",
+    "amf"
+  ],
+  "scripts": {
+    "clean": ": Nothing to clean in gatsby-transformer-webapi",
+    "test": "jest",
+    "build": ": No standalone build of the WebAPI transformer plugin in production."
```

Comment:
I don't think these scripts are needed for now. And we already setup and run jest from the root folder.